src/fsharp/FSharp.Core/FSharp.Core.nuspec

```diff
@@ -6,12 +6,16 @@
         <dependencies>
             <group targetFramework=".NETStandard2.0" />
         </dependencies>
+        <contentFiles> <!-- Deploy doc comments as Content -->
+            <files include="**/*" buildAction="None" copyToOutput="true" flatten="true" />
```

Comment:
Does this pick up anything other than the XML file?  How do we know?  I'm concerned that this will pull in a bunch of garbage.

Reply:
@brettfo, it only picks up contentFiles.  I think I can specify the exact filename though.

Reply:
There it's 'splicit now.